packages/typpx/typpx.1.4.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "1.4.2"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+homepage: "http://bitbucket.org/camlspotter/typpx"
+bug-reports: "https://bitbucket.org/camlspotter/typpx/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/typpx"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build & < "0.10" }
+  "ppxx" { >= "2.3.0" & < "2.4.0" }
+]
+available: [
+  ocaml-version >= "4.06.0" & ocaml-version < "4.07.0"
```

Comment:
Same thing as your other packages. Why this constraint ? It seem to compile fine at least on ```4.05.0```.

Reply:
Wrong tagging.  typpx.1.4.2 should be compilable only with ocaml.4.06.0.